### PR TITLE
Removing mattfarina from sig apps

### PR DESCRIPTION
**Pull request by mattfarina**
At the June 14th 2021 meeting I stepped down from SIG Apps as a
chair. It has been a pleasure to work on SIG Apps for all those
years.

/cc @kow3ns @janetkuo @soltysh

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -160,9 +160,6 @@ sigs:
     - github: kow3ns
       name: Kenneth Owens
       company: Brex
-    - github: mattfarina
-      name: Matt Farina
-      company: Rancher Labs
```
Comment (mrbobbytables) [inline sigs.yaml:165]: Thank you @mattfarina for everything <3
One thought - (you don't have to of course), but would you like to add yourself as emeritus?
https://github.com/kubernetes/community/blob/1248937700a634e36bd2548b259d755b7e7af8b9/sigs.yaml#L169-L171

File: sig-apps/README.md [hunk 2]
```diff
@@ -24,7 +24,6 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 * Janet Kuo (**[@janetkuo](https://github.com/janetkuo)**), Google
 * Kenneth Owens (**[@kow3ns](https://github.com/kow3ns)**), Brex
-* Matt Farina (**[@mattfarina](https://github.com/mattfarina)**), Rancher Labs
```
Comment (soltysh) [inline sig-apps/README.md:27]: @mattfarina don't you want to move yourself below to Emeritus Leads section?

File: sigs.yaml (hunk 1 above)
Comment (mattfarina) [reply sigs.yaml:165]: doh, missed that. I'll fix it.

File: sig-apps/README.md (hunk 2 above)
Comment (mattfarina) [reply sig-apps/README.md:27]: fixed

File: sigs.yaml (hunk 1 above)
Comment (mattfarina) [reply sigs.yaml:165]: fixed

**Comment by mrbobbytables** [comment]
/approve
Thank you again for everything @mattfarina  <3

**Pull request merged**